Modelica/Electrical/Analog/Interfaces.mo

```diff
@@ -519,9 +468,6 @@ The device temperature <strong>internalHeatPort.T</strong> can be used to descri
 
     parameter SI.Voltage offset=0 "Voltage offset";
     parameter SI.Time startTime=0 "Time offset";
-    replaceable Modelica.Blocks.Interfaces.SignalSource signalSource(
-        final offset = offset, final startTime=startTime)
-    annotation (Placement(transformation(extent={{70,70},{90,90}})));
```

Comment:
That change seems really good, but unrelated to the PR.
How did the signalSource appear?